<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaPrincipal</class>
 <widget class="QMainWindow" name="VentanaPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VentanaPrincipal</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="botonAbrirArchivo">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>320</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Abrir Archivo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonDetener">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>490</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>■</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonPausa">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>490</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>▶</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonResumir">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>490</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>⏸</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelIndividuo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Individuo X</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollAreaIndividuo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>498</width>
       <height>498</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="controlSimulacion">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>460</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Control de Simulación</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>160</y>
      <width>197</width>
      <height>140</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="labelTamCuadrado">
         <property name="text">
          <string>Tamaño de Cuadrado:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelMutacion">
         <property name="text">
          <string>Mutación (%):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelMaxPoblacion">
         <property name="text">
          <string>Máximo de Población:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelEsperanzaVida">
         <property name="text">
          <string>Esperanza de Vida:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelMaxGeneraciones">
         <property name="text">
          <string>Máximo de Generaciones:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="entradaCuadrado"/>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="entradaMutacion"/>
       </item>
       <item>
        <widget class="QSpinBox" name="entradaMaxPoblacion"/>
       </item>
       <item>
        <widget class="QLineEdit" name="entradaEsperanzaVida"/>
       </item>
       <item>
        <widget class="QLineEdit" name="entradaMaxGeneraciones"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="labelGeneracion">
       <property name="text">
        <string>Generación X</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="labelFitness">
         <property name="text">
          <string>Fitness Promedio (%): X%</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelGeneraciones">
         <property name="text">
          <string>Generaciones Efectuadas: X</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelVelocidad">
         <property name="text">
          <string>Velocidad: X</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelAceleracion">
         <property name="text">
          <string>Aceleración: X</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>botonAbrirArchivo</zorder>
   <zorder>botonDetener</zorder>
   <zorder>botonPausa</zorder>
   <zorder>botonResumir</zorder>
   <zorder>labelIndividuo</zorder>
   <zorder>scrollAreaIndividuo</zorder>
   <zorder>controlSimulacion</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
